src/routes/api/_database.js

```diff
@@ -130,6 +130,44 @@ export async function deleteAttachmentsOfCard(cardId) {
   return result;
 }
 
+//TODO: Find a way to make Mongo do the parsing and deleting work
+export async function deleteAttachmentsinFolder(userId, folderId) {
```

Comment:
Please capitalize each succeeding word in function names.

```suggestion
export async function deleteAttachmentsInFolder(userId, folderId) {
```